src/common/color_picker.c

```diff
@@ -30,6 +30,42 @@ static inline size_t _box_size(const int *const box)
   return (size_t)((box[3] - box[1]) * (box[2] - box[0]));
 }
 
+// define custom reduction operations to handle vectors of floats/ints
+// we can't return an array from a function, so wrap the array type in a struct
+typedef struct _aligned_pixel { dt_aligned_pixel_t v; } _aligned_pixel;
+typedef struct _aligned_vec_uint { DT_ALIGNED_PIXEL uint32_t v[4]; } _aligned_vec_uint;
+#define OPENMP_CUSTOM_REDUCTIONS OPENMP && !(defined(__apple_build_version__) && __apple_build_version__ < 11030000) //makes Xcode 11.3.1 compiler crash
+#ifdef OPENMP_CUSTOM_REDUCTIONS
```

Comment:
This conditional will always be true.  Did you want "#if OPENMP_CUSTOM_REDUCTIONS" instead?

Reply:
Drat, indeed! Thank you for noticing.

I also now see that I can make the custom reductions tidier. Another commit pending.